<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reader</class>
 <widget class="QMainWindow" name="reader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>DANH SÁCH ĐỘC GIẢ</string>
    </property>
    <widget class="QTableView" name="tableListReader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>661</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefreshreder">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>0</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>THÔNG TIN ĐỘC GIẢ</string>
    </property>
    <widget class="QPlainTextEdit" name="txtReaderID">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtNameReader">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtReaderSex">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>224</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtReaderStatus">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveReader">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>430</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>XÓA ĐỘC GIẢ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpdateReader">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>430</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>SỬA THÔNG TIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MÃ ĐỘC GIẢ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TÊN ĐỘC GIẢ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>234</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GIỚI TÍNH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TRẠNG THÁI THẺ</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>ĐĂNG KÍ THÔNG TIN ĐỘC GIẢ</string>
    </property>
    <widget class="QPlainTextEdit" name="txtinputName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtinputSex">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>224</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegist">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>430</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>THÊM ĐỘC GIẢ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NHẬP HỌ TÊN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>234</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NHẬP GIỚI TÍNH</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
